lib/dash/dash_parser.js

```diff
@@ -1491,6 +1503,17 @@ shaka.dash.DashParser = class {
     }
   }
 
+  /**
+   * Handles deferred releases of old SegmentIndexes
+   * content type from a previous update.
+   * @export
```

Comment:
It's not a part of public interface, don't export it

Reply:
I removed this method, as now segmentIndexes are closed by the `stream_engine`